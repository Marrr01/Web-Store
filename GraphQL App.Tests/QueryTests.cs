using Snapshooter.Xunit;
using Xunit;

namespace GraphQL_App.Tests
{
    public class QueryTests
    {
        [Fact]
        public async Task SchemaChangeTest()
        {
            var schema = await TestServices.Executor.GetSchemaAsync(default);
            schema.ToString().MatchSnapshot();
        }

        [Fact]
        public async Task GetUsers()
        {
            var result = await TestServices.ExecuteRequestAsync(
            b => b.SetQuery("{ users { login password } }"));

            result.MatchSnapshot();
        }
    }
}