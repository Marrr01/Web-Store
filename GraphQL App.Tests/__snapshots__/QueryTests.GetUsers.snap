{
  "data": {
    "users": [
      {
        "login": "usah",
        "password": "123"
      }
    ]
  }
}
